modules/gapi/samples/face_detection.cpp

```diff
@@ -0,0 +1,764 @@
+                }
+                out_faces.push_back(info);
+            }
+        }
+    }
+};// GAPI_OCV_KERNEL(RNetPostProc)
+
+GAPI_OCV_KERNEL(OCVONetPostProc, ONetPostProc) {
+    static void run(const std::vector<Face> &in_faces,
+        const std::vector<cv::Mat> &in_scores,
+        const std::vector<cv::Mat> &in_regresssions,
+        const std::vector<cv::Mat> &in_landmarks,
+        double threshold,
+        std::vector<Face> &out_faces) {
+        out_faces.clear();
+        for (unsigned int k = 0; k < in_faces.size(); ++k) {
+            const float* scores_data = (float*)in_scores[k].data;
+            const float* reg_data = (float*)in_regresssions[k].data;
+            const float* landmark_data = (float*)in_landmarks[k].data;
+            if (scores_data[1] >= threshold) {
+
+                Face info = in_faces[k];
+                info.score = scores_data[1];
+                for (int i = 0; i < 4; ++i) {
+                    info.regression[i] = reg_data[i];
+                }
+                double w = info.bbox.x2 - info.bbox.x1 + 1.0;
+                double h = info.bbox.y2 - info.bbox.y1 + 1.0;
+
+                for (int p = 0; p < NUM_PTS; ++p) {
+                    info.ptsCoords[2 * p] =
+                        info.bbox.x1 + static_cast<double>(landmark_data[NUM_PTS + p]) * w - 1;
+                    info.ptsCoords[2 * p + 1] = info.bbox.y1 + static_cast<double>(landmark_data[p]) * h - 1;
+                }
+
+                out_faces.push_back(info);
+            }
+        }
+    }
+};// GAPI_OCV_KERNEL(ONetPostProc)
+
+GAPI_OCV_KERNEL(OCVSwapFaces, SwapFaces) {
+    static void run(const std::vector<Face> &in_faces,
+        std::vector<Face> &out_faces) {
+        std::vector<Face> in_faces_copy = in_faces;
+        out_faces.clear();
+        if (!in_faces_copy.empty()) {
+            for (size_t i = 0; i < in_faces_copy.size(); ++i) {
+                std::swap(in_faces_copy[i].bbox.x1, in_faces_copy[i].bbox.y1);
+                std::swap(in_faces_copy[i].bbox.x2, in_faces_copy[i].bbox.y2);
+                for (int p = 0; p < NUM_PTS; ++p) {
+                    std::swap(in_faces_copy[i].ptsCoords[2 * p], in_faces_copy[i].ptsCoords[2 * p + 1]);
+                }
+            }
+            out_faces = in_faces_copy;
+        }
+    }
+};// GAPI_OCV_KERNEL(SwapFaces)
+
+GAPI_OCV_KERNEL(OCVTranspose, Transpose) {
+    static void run(const cv::Mat &in_mat,
+        cv::Mat &out_mat) {
+        cv::transpose(in_mat, out_mat);
+    }
+};// GAPI_OCV_KERNEL(Transpose)
+} // anonymous namespace
+} // namespace custom
+
+namespace vis {
+namespace {
+void bbox(cv::Mat& m, const cv::Rect& rc) {
+    cv::rectangle(m, rc, cv::Scalar{ 0,255,0 }, 2, cv::LINE_8, 0);
+};
+
+using rectPoints = std::pair<cv::Rect, std::vector<cv::Point>>;
+
+static cv::Mat drawRectsAndPoints(const cv::Mat& img,
+    const std::vector<rectPoints> data) {
+    cv::Mat outImg;
+    img.copyTo(outImg);
+
+    for (auto& d : data) {
```

Comment:
`for (const auto& el : data) {` for example.

Reply:
![image](https://user-images.githubusercontent.com/52420572/115589739-e1cee380-a2d8-11eb-989a-0d7605bf307d.png)
This comment about one letter variable.

Reply:
Done